Nitrox.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
-VisualStudioVersion = 16.0.30114.105
+VisualStudioVersion = 16.0.30104.148
```

Comment:
Don't know how we should handle this behavior of vs @Sunrunner37. Maybe just keep the higher number...